test/old/testdir/test_fold.vim

```diff
@@ -1555,4 +1555,19 @@ func Test_fold_screenrow_motion()
   call assert_equal(1, line('.'))
 endfunc
 
+" Make sure that when you insert indents across multiple lines with blockwise
```

Comment:
These tests are legacy tests ported from Vim and should not be changed; "our" tests are in `/test/functional`.

Reply:
Thanks. I will work on converting it.